runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -367,12 +367,12 @@
         /*
          * Total number of threads in Polling Executor Pool (shared across all listeners)
          */
-        private int streamingPollingThreadPoolSize = 2;
+        private int streamingPollingThreadPoolSize = 4;
 
         /*
          * Total number of threads in Notification Executor Pool (shared across all listeners)
          */
-        private int streamingNotificationThreadPoolSize = 4;
+        private int streamingNotificationThreadPoolSize = 8;
```

Comment:
This is a configurable parameter, the clients should  increase it as needed. It doesn't make sense as a default. The design is based on the listeners being non-blocking and this thread pool just moves items from a queue and invokes the listener.

Reply:
will make a follow up change to make it a runtime parameter.

Reply:
It just seemed easier to modify the default than seeking out the clients parameters, but I see your point
I will try to undo this change once the client values are fixed up.

Reply:
>  than seeking out the clients parameters

What does this even mean ? 


This PR has no effect, the parameters you changed will not be reflected for any client that uses the CorfuRuntimeParametersBuilder


This being the main repo, I don't think its a good idea to just try random patches.

Reply:
@Maithem good catch